### ISSUE TITLE
build qtsvgrender

### ISSUE THREAD

**Issue by JoKalliauer** [RazrFalcon/resvg-test-suite#14]
I tried 

```bash
cd ~/Documents/GitHub/resvg-test-suite/tools/qtsvgrender
g++ main.cpp
```

and got

```cmd
main.cpp:1:10: fatal error: QGuiApplication: No such file or directory
 #include <QGuiApplication>
          ^~~~~~~~~~~~~~~~~
compilation terminated.
```

**Comment by RazrFalcon** [comment]
You should use:

```
qmake
make
```

**Comment by JoKalliauer** [comment]
> You should use:
> 
> ```
> qmake
> make
> ```

If I run 
```bash
cd ~/Documents/GitHub/resvg-test-suite/tools/qtsvgrender
qmake
```

I get 
```cmd
Info: creating stash file /home/jkalliau/Documents/GitHub/resvg-test-suite/tools/qtsvgrender/.qmake.stash
Project ERROR: Unknown module(s) in QT: svg
```

**Comment by RazrFalcon** [comment]
You should install qtsvg via your distro. Looks like you are using Ubuntu, so it will be `libqt5svg5-dev`

**Comment by JoKalliauer** [comment]
Thanks that solved the issue.

How do I install wxsvgrender?

I tried
```bash
cd /home/jkalliau/Documents/GitHub/resvg-test-suite/tools/wxsvgrender
qmake
```

and get
```cmd
Usage: /usr/lib/qt5/bin/qmake [mode] [options] [files]

QMake has two modes, one mode for generating project files based on
some heuristics, and the other for generating makefiles. Normally you
shouldn't need to specify a mode, as makefile generation is the default
mode for qmake, but you may use this to test qmake on an existing project

...
...
...
```

If I run `build.sh` I get
```cmd
main.cpp:1:10: fatal error: wx/wx.h: No such file or directory
 #include <wx/wx.h>
          ^~~~~~~~~
```

**Comment by RazrFalcon** [comment]
`qmake` is only for Qt projects. For `wxsvgrender` use the `build.sh` script. Looks like on Ubuntu you need `libwxsvg-dev`.

**Comment by JoKalliauer** [comment]
> `qmake` is only for Qt projects. For `wxsvgrender` use the `build.sh` script. Looks like on Ubuntu you need `libwxsvg-dev`.

`sudo apt-get install -y libwxsvg-dev`
results in
```cmd
Package libwxsvg-dev is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libwxsvg-dev' has no installation candidate
```

However I downloaded https://sourceforge.net/projects/wxsvg/ and run
```bash
cd ~/prgm/wxsvg-1.5.22
./configure
make
make install
```

**Comment by RazrFalcon** [comment]
Yes, looks like it became available only since 19.04.
